compliance_checker/tests/test_acdd.py

```diff
@@ -395,20 +373,6 @@ def test_variables(self):
         for result in results:
             self.assert_result_is_bad(result)
 
-    def test_acknowledgement(self):
```

Comment:
I removed this method since the "acknowledgement" attribute is a global attribute. While it does have its own method implemented in `acdd.py`, the method only serves to check if either/both the two possible spelling variations exist in the dataset ("acknowledgment" vs "acknowledg __e__ ment" with the added "e"). What say you, @Bobfrat @benjwadams?

Reply:
gotcha, my bad, we're in the tests not the checks. I was looking at line 46 and got confused:
```
"acknowledgment/acknowledgement",
```